tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply_test.py

```diff
@@ -0,0 +1,76 @@
+"""
+Integration test for DrugRefillsDaysSupply module
+Original Issues: DC-403, DC-815
+"""
+# Python Imports
+import os
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.drug_refills_days_supply import DrugRefillsDaysSupply
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+from common import DRUG_EXPOSURE, JINJA_ENV
+
+# Third party imports
+
+
+class DrugRefillsDaysSupplyTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        cls.project_id = os.environ.get(PROJECT_ID)
+        cls.dataset_id = os.environ.get('COMBINED_DATASET_ID')
+        cls.sandbox_id = f'{cls.dataset_id}_sandbox'
+
+        cls.rule_instance = DrugRefillsDaysSupply(cls.project_id,
+                                                  cls.dataset_id,
+                                                  cls.sandbox_id)
+
+        cls.fq_table_names = [
+            f'{cls.project_id}.{cls.dataset_id}.{DRUG_EXPOSURE}'
+        ]
+
+        cls.fq_sandbox_table_names = [
+            f'{cls.project_id}.{cls.sandbox_id}.{cls.rule_instance.sandbox_table_for(DRUG_EXPOSURE)}'
+        ]
```

Comment:
This works, but doesn't cause the error to surface with the `affected_tables` variable when that isn't set correctly.  You can accomplish the same thing with the following and you don't have to "tweak" this code as often.  This also lets you the import of `DRUG_EXPOSURE`.
```suggestion
        cls.fq_sandbox_table_names = []
        for table in cls.rule_instance.get_sandbox_tablenames():
            cls.fq_sandbox_table_names.append(f'{cls.project_id}.{cls.sandbox_id}.{table}')
```